charts/opentelemetry-demo/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 type: application
 name: opentelemetry-demo
-version: 0.36.1
+version: 0.36.2
```

Lately, the merging order has been tricky for the demo chart updates. One of my changes was merged in first and used version [0.63.3](https://github.com/open-telemetry/opentelemetry-helm-charts/releases/tag/opentelemetry-demo-0.36.3) (which was unexpected but manageable). @rlankfo can you bump this to 0.36.4 and generate the examples? This way we get the correct latest tag applied to the correct chart release, we would skip 0.36.2.
```suggestion
version: 0.36.4
```

---

done! thanks @jvoravong